<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Debug</class>
 <widget class="QDialog" name="Debug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug</string>
  </property>
  <widget class="QPushButton" name="onDebug">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始调试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onCon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>继续</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onRun">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="bugMes">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="onNext">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="varEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addWatch">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>16</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>add watch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onStep">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>step</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
